cmake/Scripts/configure

```diff
@@ -48,6 +48,7 @@ Optional Packages:
   --with-superlu=PATH     user defined path to SuperLU library
   --with-umfpack=PATH     use UMFPACK/SuiteSparse from a specified location
   --with-ert=PATH         Use ERT libraries
+  --with-zoltan=PATH      Use ZOLTAN libraries
```

Comment:
Usually path is required to be /usr/local and then it should contain include and lib subdirectories. 
Some packages, unfortunately, deviate from that standard. If the packages uses autotools then its same to assume that it will be like described above.  I hope this is also the case with cmake.